🗂️ hub.md

```diff
@@ -9,20 +9,3 @@ publish: true
 # 🗂️ hub
 
 #placeholder/description 
-
```

Comment:
I would delete this note.

Reply:
Done - e469086461ac99f428aac65ace9ee2e195582be4